entrypoints/apply-shim.js

```diff
@@ -36,18 +36,21 @@ class ApplyShimInterface {
       return;
     }
     this.customStyleInterface = window.ShadyCSS.CustomStyleInterface;
-    if (this.customStyleInterface) {
-      this.customStyleInterface['transformCallback'] = (style) => {
-        applyShim.transformCustomStyle(style);
-      };
-      this.customStyleInterface['validateCallback'] = () => {
-        requestAnimationFrame(() => {
-          if (this.customStyleInterface['enqueued']) {
-            this.flushCustomStyles();
-          }
-        });
-      }
+    if (!this.customStyleInterface) {
+      this.customStyleInterface = new CustomStyleInterface();
```

Comment:
Seems like it might make sense to have an import that can bring in this singleton instance rather than this slightly tricky conditional here?

Reply:
Yep, will do!